portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="contact.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="contact_content">
```

Comment:
Use hyphens instead of underscores for class names and IDs.  So... contact-content here =)
https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Reply:
Moved all prev code to add footer